assets/js/phoenix_live_view/rendered.js

```diff
@@ -17,6 +17,66 @@ import {
   isCid,
 } from "./utils"
 
+export let modifyRoot = (html, attrs, innerHTML) => {
+  html = html.trimStart()
+  let tagStartsAt = null
+  let pos = 0
+  while(pos < html.length){
+    let maybeStart = html.indexOf("<", pos)
+    if(maybeStart === -1){ break }
```

Comment:
Should you throw instead? Otherwise `html.slice()` later will fail in confusing ways.